ruby/ruby-install/Portfile

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        postmodern ruby-install 0.7.0 v
+categories          ruby
+license             MIT
+platforms           darwin
+supported_archs     noarch
+
+maintainers         {catlett.info:chad @chadcatlett} openmaintainer
+
+description         Installs Ruby, JRuby, Rubinius, TruffleRuby or mruby.
+long_description    ${description}
+
+checksums           rmd160  dcd7edba9588c71a51a2416b7277b334f27efd0e \
+                    sha256  2e0c6db77cdf50d30f68db3c10cda72bab6b6d4be12cc11aec6fcfad8f99911f \
+                    size    28545
+
+use_configure       no
+
+build               {}
+
+destroot.cmd        make
+destroot.target     install
```

Comment:
Those are the default values for destroot.cmd and destroot.target.

Reply:
fixed